tasks/debian10.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Install Pacemaker cluster packages to all nodes
+  apt:
+    name: "{{ cluster_node_is_remote | bool | ternary(pacemaker_remote_packages, pacemaker_packages) }}"
+    state: 'present'
+
+- name: Check if Corosync configuration is default configaration
+  shell: '/usr/bin/dpkg --verify corosync'
+  register: result
```

Comment:
(medium) this task reports always 'changed', as we are only getting the output from here could you please add `changed_when: False` to suppress the 'changed'?

(medium) in `--check` mode this tasks is not run making the next task failed on undefined variables. As this is just fetching the output we can instruct ansible to run this in `--check` mode with `check_mode: False` so we always run the 'dpkg --verify corosync' command

Reply:
this is also resolved.